cmake/dependencies/PICSAR.cmake

```diff
@@ -39,17 +39,12 @@ function(find_picsar)
     endif()
 endfunction()
 
-if(WarpX_QED)
-    option(WarpX_picsar_internal   "Download & build PICSAR" ON)
-    set(WarpX_picsar_repo "https://github.com/ECP-WarpX/picsar.git"
-        CACHE STRING
-        "Repository URI to pull and build PICSAR from if(WarpX_picsar_internal)")
-    set(WarpX_picsar_branch "development"
-        CACHE STRING
-        "Repository branch for WarpX_picsar_repo if(WarpX_picsar_internal)")
+option(WarpX_picsar_internal   "Download & build PICSAR" ON)
+set(WarpX_picsar_repo "https://github.com/lucafedeli88/picsar.git"
+CACHE STRING
+    "Repository URI to pull and build PICSAR from if(WarpX_picsar_internal)")
+set(WarpX_picsar_branch "improve_makefile"
```

Comment:
I will modify this as soon as  https://github.com/ECP-WarpX/picsar/pull/7 is merged